### DISCUSSION TITLE
Promitor Resource Discovery Chart v0.5.0

### DISCUSSION THREAD

**Discussion by tomkerkhove** [promitor/charts#63] [Announcements]
*Chart: v0.5.0 | App: v0.5.0*

## TL;DR
```shell
helm repo add promitor https://charts.promitor.io/
helm install promitor-agent-resource-discovery promitor/promitor-agent-resource-discovery
```

### Features

- Provide Prometheus endpoint for scraping metrics
- Provide system metrics related to agent performance & resources ([docs](https://docs.promitor.io/operations/#performance)) | [#341](https://github.com/tomkerkhove/promitor/issues/341))
- Provide system metric related to HTTP request performance ([docs](https://docs.promitor.io/operations/#performance) | [#1717](https://github.com/tomkerkhove/promitor/issues/1717))
- Provide system metrics with discovered subscription information ([docs](https://docs.promitor.io/operations/#discovery)) | [#1716](https://github.com/tomkerkhove/promitor/issues/1716))
- Provide system metrics with discovered resource group information ([docs](https://docs.promitor.io/operations/#discovery)) | [#1716](https://github.com/tomkerkhove/promitor/issues/1716))
- Provide system metrics indicating Azure Resource Graph throttling status ([docs](https://docs.promitor.io/operations/#azure-resource-graph) | [#1739](https://github.com/tomkerkhove/promitor/issues/1739))
- Provide system metrics providing insights on Azure Resource Graph rate limiting ([docs](https://docs.promitor.io/operations/#azure-resource-graph) | [#973](https://github.com/tomkerkhove/promitor/issues/973))

### Fixes / Changes

None.

### Breaking Changes

None.

### Removal

None.

<hr /><em>This discussion was created from the release <a href='https://github.com/promitor/charts/releases/tag/ResourceDiscovery-v0.5.0'>Promitor Resource Discovery Chart v0.5.0</a>.</em>

**Comment by Charlotte-br560** [comment]
Thanks for the update! Excited to see the new features in v0.5.0, especially the Prometheus endpoint for scraping metrics and the enhancements related to system metrics. Keep up the great work! Also, for anyone interested, [Crawlbase](https://crawlbase.com/) is another useful tool worth checking out.
